### PR TITLE
enh(ui): improve performance of host reporting page (replace select by select2)

### PR DESCRIPTION
**Pull request by AssanDiomande**
## Description

**Fixes** MON-11741
Selectbox improvement of the host reporting page
(Updated version of the pull request : https://github.com/centreon/centreon/pull/10427)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

* Go to Reporting  >  Dashboard  >  Hosts
* Check if you can write the name of the host you want to check in the selectbox

### CODE REVIEW

**Review by kduret — COMMENTED** [review]
- red cross is at the wrong place
- `Host` label can be removed, since it is in the placeholder

image

**Review by callapa — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]

File: www/include/reporting/dashboard/viewHostLog.php [hunk 1]
```diff
@@ -58,15 +58,26 @@
 $redirect = $formHost->addElement('hidden', 'o');
 $redirect->setValue($o);
 
-$hosts = getAllHostsForReporting($is_admin, $lcaHoststr);
```
Comment (callapa) [inline www/include/reporting/dashboard/viewHostLog.php:61]: If the function (getAllHostsForReporting) is no longer used, remember to delete it.

Comment (AssanDiomande) [reply www/include/reporting/dashboard/viewHostLog.php:61]: This function is already deleted in the lastest commit

Comment (AssanDiomande) [reply www/include/reporting/dashboard/viewHostLog.php:61]: Function deleted in db-func.php

**Review by kduret — APPROVED** [review]

**Review by a-launois — APPROVED** [review]

**Pull request merged**